desktop/cmp/form/JsonField.js

```diff
@@ -47,6 +47,14 @@ export class JsonField extends HoistField {
         editorProps: PT.object
     };
 
+    constructor() {
+        super();
+        this.addReaction({
+            track: () => XH.darkTheme,
+            run: () => this.editor.setOption('theme', XH.darkTheme ? 'dracula' : 'default')
```

Comment:
Let's just make this null safe -- editor may be null, not setup immediately

Reply:
Can we also cleanup the editor in destroy() instead of willUnmount()? (see below)